raft/src/test/java/org/apache/kafka/raft/KafkaNetworkChannelTest.java

```diff
@@ -288,7 +290,7 @@ private ApiMessage buildTestRequest(ApiKeys key) {
                 return VoteRequest.singletonRequest(topicPartition, clusterId, leaderEpoch, leaderId, lastEpoch, 329);
 
             case FETCH:
-                FetchRequestData request = RaftUtil.singletonFetchRequest(topicPartition, topicId, fetchPartition ->
+                FetchRequestData request = FetchRpc.singletonFetchRequest(topicPartition, topicId, fetchPartition ->
```

Comment:
I wouldn't be upset having both Fetch and FetchSnapshot in the same class. But I guess same argument could be made for Begin and End Quorum (and I see how naming might get more difficult)